test/Concurrency/Runtime/actor_deinit_escaping_self.swift

```diff
@@ -0,0 +1,69 @@
+// RUN: %target-run-simple-swift( -Xfrontend -disable-availability-checking -parse-as-library)
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// REQUIRES: concurrency_runtime
+// UNSUPPORTED: back_deployment_runtime
+
+import _Concurrency
+@preconcurrency import Dispatch
+import StdlibUnittest
+
+actor EscapeLocked {
+  var k: Int = 1
+  
+  func increment() {
+    k += 1
+  }
+  
+  deinit {
+    let g = DispatchGroup()
+    g.enter()
+    Task.detached {
+      await self.increment()
+      g.leave()
+    }
+    let r = g.wait(timeout: .now() + .milliseconds(500))
+    expectEqual(r, .timedOut)
+    expectCrashLater(withMessage: "Assertion failed: (!oldState.getFirstJob() && \"actor has queued jobs at destruction\"), function destroy")
```

Comment:
I'm not sure if that is even valid to test for assertions in stdlib as an expected behaviour. If stdlib is build without assertions, test will fail